app/models/configuration.rb

```diff
@@ -0,0 +1,49 @@
+require "net/http"
+require "uri"
+require "json"
+
+class Configuration
+  attr_reader :errors
+  attr_accessor :customer_id, :api_key
+
+  def initialize(okapi)
+    @errors = {}
+    @okapi = okapi
+  end
+
+  def load!
+    response = @okapi.user.get '/configurations/entries?query=module=KB_EBSCO'
+    response["configs"].each do |config|
+      params = Rack::Utils.parse_query(config["value"])
+      self.customer_id = params["customer-id"]
+      self.api_key = params["api-key"]
+    end
+   response
+  end
+
+  def id
+    'rmapi'
+  end
+
+  def valid?(arg)
+    urlstr = "https://sandbox.ebsco.io/rm/rmaccounts/#{@customer_id}/vendors?search=zz12&offset=1&orderby=vendorname&count=1"
```

Comment:
i'm guessing we're going to continue using an ENV var for the RMAPI address for the time being.  we should probably use that here instead of hardcoding